gemfiles/rails_70/Gemfile

```diff
@@ -16,6 +16,9 @@ group :rails do
   gem 'combustion'
   gem 'capybara'
 
+  # FIXME: relax this dependency when Ruby 3.0 support will be dropped
```

Comment:
Ruby 3.0 support was already dropped on default branch so this shouldn’t be necessary.